Docs remove conda (#2516)
GMLC-TDC/HELICS: docs/user-guide/installation/package_manager.md

# Package Manager

## Install using pip (Windows, macOS, Linux, other)

Install Python with pip. Upgrade pip to a recent version using `python -m pip install --upgrade`.

If you're on a supported version of Windows, macOS, or Linux (see the [HELICS PyPI page](https://pypi.org/project/helics/) for details) you can then use `pip` to install the HELICS Python interface and helics-apps.

```bash
pip install 'helics'
```

or

```bash
pip install 'helics[cli]'
```

The second version with the "[cli]" additionally installs additional tools that provides an easy method for launching co-simulations that is used in the [HELICS User Guide Examples](../examples/examples_index.md) and is recommended.

> If you are on an unsupported OS or Python version, you will need to install a copy of HELICS first.

Reviewer: ```suggestion
If you are on an unsupported OS or CPU architecture, you may need to install a copy of HELICS first.
```
